packages/core/stories/toolkit-provider.doc.stories.mdx

```diff
@@ -96,6 +96,117 @@ It requires no configuration, but is included in your app when you use a Toolkit
 AriaAnnouncer is ever required, only the outermost instance of ToolkitProvider will instantiate the AriaAnnouncer. If you use
 nested ToolkitProviders, these will not inject further instances of AriaAnnouncer.
 
+## useDensity
+
+You can use `useDensity` hook to access the currently active density of your app in the provided scope.
+
+### API
+
+```js
+import { useDensity } from "@jpmorganchase/uitk-core";
+```
+
+### Example use
+
+Use density hook returns the string value of the currently set density in scope. You can optionally override the value bu passing your own value as the first parameter to the function.
+
+```tsx
+import { useDensity } from "@jpmorganchase/uitk-core";
+import cx from "classnames";
+
+function MyComponent() {
+  const [override, setDensityOverride] = useState();
+  const density = useDensity(override);
+  return (
+    <>
+      <div
+        className={cx("myComponent", {
+          "myComponent--touch": density === "touch",
+        })}
+      >
+        Hello World!
+      </div>
+      <button
+        onClick={() => {
+          setOverride("touch");
+        }}
+      >
+        Set Touch Density
+      </button>
+    </>
+  );
+}
+```
+
+## useTheme
+
+You can use `useTheme` hook to access the value of the currently active themes.
```

Comment:
Rewrite - You can use **the** 'useTheme' hook to access the value which indicates which themes are currently active.